src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -994,9 +1000,9 @@ public BriefcaseFormDefinition getFormDefinition() {
   public FilesSkipped totalFilesSkipped() {
     //Determine if all files where skipped or just some
     //Note that if totalInstances = 0 then no files were skipped
-    if (totalIntances == 0) {
+    if (totalInstances == 0 || totalFilesSkipped == 0) {
       return FilesSkipped.NONE;
     }
-    return totalFilesSkipped == totalIntances ? FilesSkipped.ALL : FilesSkipped.SOME;
+    return totalFilesSkipped == totalInstances ? FilesSkipped.ALL : FilesSkipped.SOME;
```

Comment:
i think we could avoid using the ternary operator since i could not find any other use case in the code base and it could be confusing at times